README.md

```diff
@@ -0,0 +1,82 @@
+# Kartenforum Georeference
+
+This repository contains the code for deploying a georeference service as well as a daemon for an asynchrone permanent update of the georeference data basis.
+
+## Install
+
+The tools are developed with Python 3.9.7. They rely on the following dependencies:
+
+```
+apt-get install python3 python3-virtualenv libpq-dev gdal-bin libgdal-dev gcc uwsgi
+```
+
+The python dependencies are installed within a virtual environment. Therefor run the following commands:
+
+```
+virtualenv python_env
+./python_env/bin/pip install -e ".[testing]"
+```
+
+## Testing & Development
+
+For local development and testing of the project a few further steps have to be executed. First download the `test_data.tar.xz` unpack it and place the image within the directory `./tmp/org_new/`. The password for the SFTP-Service is `8C2Kdpxc2lpUoYBX`.
+
+```
+# Download the images
+scp u279620-sub2@u279620-sub2.your-storagebox.de:/test_data.tar.xz ./
+tar -xf test_data.tar.xz -C ./tmp/org_new/
```

Comment:
```suggestion
mkdir /tmp/org_new/
tar -xf test_data.tar.xz -C /tmp/org_new/
```
The directory `/tmp/org_new/` must exist. Otherwise the `tar` command will fail.